riotfile.py

```diff
@@ -81,6 +81,14 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         "hypothesis": latest,
     },
     venvs=[
+        Venv(
```

Comment:
why does `riot` need `pre-commit`?

Reply:
This is just a riot command for devs to be able to install `pre-commit` and initialize the configuration setup which will automate the hooks on `git commit`.

Reply:
this will install it into a virtualenv, which won't be available to `git` at commit time ?

Reply:
This `pre-commit install` step just installs a script to `.git/hooks/pre-commit` which automates the pre-commit hook at commit time, so it should be fine to be in a virtualenv (just tested myself) 😄